styles/aboutus.css

```diff
@@ -0,0 +1,164 @@
+:root {
+    --background-color: #FAF4E8;
+    --feature-color: #34959E;
+    --footer-color: #9D6034;
+    --nav-bar: #DAA46D;
+    --text-color: #382B22;
+}
+
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    background-color: var(--background-color);
+    font-family:Arial, Helvetica, sans-serif;
+    height: 90%;
+    margin: 0;
+    padding: 0;
+    width: 100%;
+}
+
+h2 {
+    margin: 0;
+}
+
+h3 {
+    margin: 0;
+}
+p {
+    margin: 0;
+}
+
+.header {
+    
+}
+
+.nav-bar {
+    /* margin: 32px 0 0 32px; */
```

Comment:
If you don't intend to use the outcommented code then maybe get rid of it in the future?